### DISCUSSION TITLE
Just wondering what is halfwidth in Crystallography

### DISCUSSION THREAD

**Discussion by Jinhry** [mtex-toolbox/mtex#1634] [Ask Anything]
Dear All,

I just started to show interest in crystallography and was wondering if someone could explain what is half-width that I see used in ODF calculations but no real explanation was found... I guess it must be common knowledge but could you help me a bit here? Highly appreciate your help!

Best,

**Comment by ralfHielscher** [comment]
Have a look at https://mtex-toolbox.github.io/DensityEstimation.html.

Ralf.
